README.md

```diff
@@ -7,95 +7,66 @@
 [![Total Downloads][ico-downloads]][link-downloads]
 [![SensioLabs Insight][ico-sensiolabs]][link-sensiolabs]
 
-Middleware to execute a handler if the response returned by the next middlewares has any error (status code 400-599) or throws a `Middlewares\HttpErrorException`.
-
-This package provides the `Middlewares\HttpErrorException` that you can use to send context-related data to the error handler. The methods `setContext(array $data)` and `getContext()` methods allows to assign and retrieve the error context data used in the error handler.
+Middleware to catch and format errors encountered while handling the request.
 
 ## Requirements
 
-* PHP >= 7.0
+* PHP >= 7.1
 * A [PSR-7 http library](https://github.com/middlewares/awesome-psr15-middlewares#psr-7-implementations)
 * A [PSR-15 middleware dispatcher](https://github.com/middlewares/awesome-psr15-middlewares#dispatcher)
 
 ## Installation
 
 This package is installable and autoloadable via Composer as [middlewares/error-handler](https://packagist.org/packages/middlewares/error-handler).
 
-```sh
+```shell
 composer require middlewares/error-handler
 ```
 
 ## Example
 
 ```php
-use Interop\Http\Server\RequestHandlerInterface;
-use Psr\Http\Message\ResponseInterface;
-use Psr\Http\Message\ServerRequestInterface;
-
-class ErrorRequestHandler implements RequestHandlerInterface
-{
-    public function handle(ServerRequestInterface $request): ResponseInterface
-    {
-        //Get the error info as an instance of Middlewares\HttpErrorException
-        $error = $request->getAttribute('error');
-
-        //The error can contains context data that you can use, for example for PSR-3 loggin
-        Logger::error("There's an error", $error->getContext());
-
-        //Any output is captured and added to the response's body
-        echo $error->getMessage();
-
-        return (new Response())->withStatus($error->getCode());
-    }
-}
-
+use Middlewares\ErrorHandler;
+use Middlewares\Formatter;
```

Comment:
I think `Middlewares\Formatter` should be moved to `Middlewares\ErrorHandler\Formatter`, to avoid possible conflicts with other middlewares' packages

Reply:
Makes sense. 👍